README.md

```diff
@@ -12,53 +12,108 @@ Below documentation is meant for manual testing and additional details. It's alr
 The only requirement is Docker. All the libraries and packages are built in the docker image.
 If for some reason one needs to setup the requirements on the host system see the Dockerfile.
 
-## Credentials setup
+## Getting started
 
-### Amazon Web Services
+This is intended to be used as fast setup recipe, for additional details read the documentation below.
 
-Yous should make sure [AWS CommandLine interface credentials are set](http://docs.ansible.com/ansible/latest/intro_dynamic_inventory.html#example-aws-ec2-external-inventory-script)
-either by environment variables or `~/.aws/credentials` file.
-
-If you have configured multiple AWS credentials you can pass AWS_PROFILE variable before the commands:
+Setup Vault authentication:
 
 ```bash
-AWS_PROFILE=aeternity ansible-inventory --list
+export VAULT_ADDR=https://the.vault.address/
+export VAULT_GITHUB_TOKEN=your_personal_github_token
+```
+
+Run the container:
+
+```
+docker pull aeternity/infrastructure
+docker run -it --env-file env.list aeternity/infrastructure
```

Comment:
It is a bit unclear from where the `docker run` command should be ran. The assumption is that it should be ran from within a copy of the infrastructure repository. However, reading the documentation and following the steps, one might get carried away, and just follow the steps without cloning. It should be clear that `docker run` must be executed from a copy of the infrastructure repository, so the env.list file is present. 

My suggestion is to add the following two steps before the `docker run` command, so it is obvious. Especially to new team members:

`git clone git@github.com:aeternity/infrastructure.git`
`cd infrastructure`

Either that, or steps to create the env.list file manually.

Reply:
Good point, I forgot to work on that issue. I updated the commands in ce963ae